NuGet/OpenRiaServices.T4.nuspec

```diff
@@ -16,8 +16,14 @@
     <language>en-US</language>
     <tags>WCF RIA Services RIAServices T4 OpenRiaServices</tags>
     <dependencies>
-      <dependency id="OpenRiaServices.Server" version="$version$" />
-      <dependency id="Mono.Cecil" version="0.11.5" />
+      <group targetFramework="net6.0">
+        <dependency id="OpenRiaServices.Server" version="$version$" />
+        <dependency id="Mono.Cecil" version="0.11.5" />
```

Comment:
For net6 we need to go through all dependencies of Tools and Tools.TextTemplate projects and add those here (including System.ComponentModel.* and related )

Reply:
not a blocker: can be done later